pkg/server/plugin/upstreamauthority/certmanager/internal/v1/types_certificaterequest.go

```diff
@@ -49,7 +49,7 @@ const (
 // point in time request for a certificate and cannot be re-used.
 type CertificateRequest struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+	metav1.ObjectMeta `json:"metadata"`
```

Comment:
This file is copied from `cert-manager` because they discourage importing from their module. We should avoid making modifications to this, so could you revert the change in this file? 

This pattern seems to be common in general with k8s, see for example https://pkg.go.dev/k8s.io/api@v0.27.2/core/v1#Pod